trk/src/Screens/ClimbCreate/index.js

```diff
@@ -62,6 +62,18 @@ const ClimbInputData = (props) => {
     console.log('Cancel button was pressed in AndroidPrompt');
   };
 
+  const validateInput = () => {
+    if (name.length > 10) {
```

Comment:
@redpepper-nag : 10 characters limit is an experiment. Do ask Eddie what he thinks of it (by testing it)